hicexplorer/readBed.py

```diff
@@ -113,7 +116,7 @@ def get_bed_interval(self, bed_line):
 
         >>> bed_line="chr1\t0\t1000\tgene_1\t0.5\t-\t0\t1000\t0\t3\t10,20,100\t20,200,700"
         >>> with open('/tmp/test.bed', 'w') as fh:
-        ...     fh.write(bed_line)
+        ...     foo = fh.write(bed_line)
```

Comment:
'write' should return the number of written bytes in Python2 and 3. But Python 2 does not return something, Python 3 does. Without 'foo = ' the doc string test with Python 3 is failing.